src/Test/contconic.jl

```diff
@@ -225,17 +203,16 @@ function lin3test(model::MOI.ModelLike, config::TestConfig)
     if config.solve
         MOI.optimize!(model)
 
-        @test MOI.canget(model, MOI.TerminationStatus())
         if config.infeas_certificates
             @test MOI.get(model, MOI.TerminationStatus()) == MOI.Success
         else
             @test MOI.get(model, MOI.TerminationStatus()) in [MOI.InfeasibleNoResult, MOI.InfeasibleOrUnbounded]
         end
-        if MOI.canget(model, MOI.PrimalStatus())
-            @test MOI.get(model, MOI.PrimalStatus()) == MOI.InfeasiblePoint
+        if config.infeas_certificates || MOI.get(model, MOI.ResultCount()) > 0
```

Comment:
The 2 "or" statements here are a bit confusing: `config.infeas_certificates || MOI.get(model, MOI.ResultCount()) > 0`
and `MOI.get(model, MOI.PrimalStatus()) in (MOI.NoSolution, MOI.InfeasiblePoint)`.

Could you untangle what is expected in what cases and/or add a comment about why these different cases can happen?

Doesn't `config.infeas_certificates` imply that we expect `MOI.get(model, MOI.ResultCount()) > 0` and PrimalStatus to be `NoSolution`?

Same comment applies right below.